libs/reactivex/Observable_MinOperator.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+
+namespace Cusco.ReactiveX;
+
+public abstract partial class Observable<T>
+{
+  public Observable<T> Min(IComparer<T> comparer)
```

Comment:
Could add the same method but taking a `Comparison<T>` as a parameter, this way we cover all comparison APIs from the standard library 😊